### ISSUE TITLE
Private passphrase showing digits on terminal

### ISSUE THREAD

**Issue by rodrigondec** [matheusd/dcr-split-ticket-matcher#67]
The `splitticketbuyer` CLI asks for my private wallet passphrase.

The input on terminal for the passprhase doesn't hide my input digits
![image](https://user-images.githubusercontent.com/3301060/69115743-768c3c80-0a68-11ea-94f0-736190d6a4cc.png)

**This goes against every security principle I know!**

**Comment by matheusd** [comment]
I was pretty sure I documented this somewhere, but seems that I didn't.

This follows the same pattern as for unlocking the wallet for CLI users, so you need to use the `promptsecret` app (see the [docs for unlocking dcrwallet](https://docs.decred.org/wallets/cli/dcrctl-basics/#unlocking-your-wallet) when running (`promptsecret | splitticketbuyer ...`).

But anyway, I'll see to more explicitly call that out and bring in echo off.

**Comment by rodrigondec** [comment]
Yes, I followed the [oficial client CLI docs](https://github.com/matheusd/dcr-split-ticket-matcher/blob/master/docs/client-cli.md). And it didn't mention the `promptsecret`. 

Ty

**Comment by rodrigondec** [comment]
Found the docs with instructions on [manual configuration docs](https://github.com/matheusd/dcr-split-ticket-matcher/blob/master/docs/config-client.md).

Didn't access this link on my first configuration. Should it be on another page as well?

**Comment by chen-ky** [comment]
I wrote a bash script which will read from stdin for your password without revealing it in the terminal. The script is available [here](https://github.com/chen-ky/decred-splitticketbuyer-script). The script will also automatically retry buying tickets if previous attempt failed.
